src/app/routes/api/v3/admin/forms/__tests__/admin-forms.submissions.routes.spec.ts

```diff
@@ -1319,6 +1320,7 @@ describe('admin-form.submissions.routes', () => {
         metadata: [
           {
             number: 1,
+            payments: null,
```

Comment:
Shall we add UT for when payments is not null as well? Easier to add the UT if the payment meta creation is a function by itself.

Reply:
9bd139ea1